src/services/ticket.ts

```diff
@@ -42,24 +41,7 @@ async function updateTicket(originalTicket: Ticket, updateRequestTicket: Partial
   return resultTicket;
 }
 
-export async function editTicketById(
-  req: FastifyRequest<{
-    Params: { ticketId: string };
-    Body: Ticket;
-  }>,
-  res: FastifyReply,
-): Promise<Ticket> {
-  const ticketIdQuery = req.params.ticketId;
-  const editRequestBody = req.body;
-
-  const retrievedTicket = await getTicketById(ticketIdQuery);
-  const updatedTicket = await updateTicket(retrievedTicket, editRequestBody);
-
-  return res.send(updatedTicket);
-}
-
```

Comment:
Moved the logic to the controller layer. The same change applied to project service too.